README.md

```diff
@@ -6,15 +6,26 @@ Goal is to use 100% (or close to it) Elm on the front-end and Elixir w/ [Phoenix
 
 ## Development
 
-To install prerequisites run:
+Prerequisites
+  - Elixir v1.3
```

Comment:
Let's add Elm to the list of prerequisites and link both Elixir and Elm to their respective websites.